coalib/coala.py

```diff
@@ -59,11 +60,34 @@ def main(debug=False):
 
         if args.show_bears:
             from coalib.settings.ConfigurationGathering import (
-                get_filtered_bears)
-
-            local_bears, global_bears = get_filtered_bears(
-                args.filter_by_language, log_printer)
-
+                get_all_bears)
+            filtered_bears = get_all_bears(log_printer)
+            if args.filter_by_language:
+                # filter_by_language option should be deprecated soon.
+                # Since it is already covered by filter_by option.
+                log_printer.warn(
+                    "'--filter-by-language ...' option is deprecated. "
+                    "Use '--filter-by language ...' option instead.")
+                args.filter_by = [] if args.filter_by is None else \
+                    args.filter_by
+                args.filter_by.append(['language'] + args.filter_by_language)
```

Comment:
> Regarding, `+=`, it actually creates new object.

Nope it doesn't, it's in-place :)

```
>>> a=[1,2,3]
>>> id(a)
140015517382472
>>> a+=[4,5]
>>> id(a)
140015517382472
```

But okay so having a twice-nested list is intended. Does argparse create the same, e.g. when doing
```
--filter-by language python C --filter-by can_fix all
```
Does it create `[['language', 'python', 'C'], ['can_fix', 'all']]`?